eslint.config.js

```diff
@@ -51,6 +50,11 @@ export default tseslint.config(
         ...globals.node,
       },
     },
+    settings: {
+      node: {
+        version: '^18.0.0 || ^20.0.0 || >=22.0.0',
+      },
+    },
```

Comment:
`packages/vite/src/node/__tests__/package.json` was picked up for `packages/vite/src/node/__tests__/build.spec.ts` by eslint-plugin-n and because that package.json does not have `engines` field, `>=16.0.0` was used and an lint error happened for `util.stripVTCharacters`.
https://github.com/eslint-community/eslint-plugin-n?tab=readme-ov-file#configured-nodejs-version-range
I set this to make `^18.0.0 || ^20.0.0 || >=22.0.0` to be applied across the repo.